oshi-core/src/main/java/oshi/hardware/platform/linux/LinuxCentralProcessor.java

```diff
@@ -58,43 +58,44 @@ public LinuxCentralProcessor() {
         LOG.debug("Initialized Processor");
     }
 
+
     private void initVars() {
         String[] flags = new String[0];
         List<String> cpuInfo = FileUtil.readFile("/proc/cpuinfo");
         for (String line : cpuInfo) {
-            String[] splitLine = line.split("\\s+:\\s");
+            String[] splitLine = ParseUtil.whitespaceCloneWhitespace.split(line);
             if (splitLine.length < 2) {
                 break;
             }
             switch (splitLine[0]) {
-            case "vendor_id":
-                setVendor(splitLine[1]);
-                break;
-            case "model name":
-                setName(splitLine[1]);
-                break;
-            case "flags":
-                flags = splitLine[1].toLowerCase().split(" ");
-                boolean found = false;
-                for (String flag : flags) {
-                    if ("lm".equals(flag)) {
-                        found = true;
-                        break;
+                case "vendor_id":

```

Comment:
This looks like mostly formatting changes. Please use the [same Eclipse formatting](https://github.com/oshi/oshi/blob/master/oshi-dist/src/main/resources/eclipse/format.xml) as the project standard.